src/main/java/core/basesyntax/ConstructFigure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public interface ConstructFigure {
+
+    abstract double getArea();
+
+    abstract String getName();
+
+    abstract String getInfo();
```

Comment:
dont invent your own names - it's required to be `draw()`. also should not have return type
```suggestion
    abstract void draw();
```